test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -987,9 +998,9 @@ void UcxConnection::request_completed(ucx_request *r)
     ucs_list_del(&r->pos);
 
     if (_disconnect_cb != NULL) {
-        UCX_CONN_LOG << "completing request " << r << " with status \""
-                     << ucs_status_string(r->status) << "\" (" << r->status
-                     << ")" << " during disconnect";
+        UCX_CONN_LOG << "completing request " << r << " (" << r->what
+                     << ") with status \"" << ucs_status_string(r->status)
```

Comment:
maybe it should be like this:
"completing ucp_tag_recv_nb request 0x12345 with status 'Connection reset' during disconnect"